com.chartboost.mediation.demo/Assets/Demo/Demo.cs

```diff
@@ -242,48 +235,106 @@ public void OnCreateBannerClick()
             6 => ChartboostMediationBannerAdScreenLocation.BottomRight,
             _ => ChartboostMediationBannerAdScreenLocation.TopCenter
         };
-        _bannerAd.Load(screenPos);
+        
+        Log($"Creating new Unity banner ad");
+        _bannerAd = ChartboostMediation.GetUnityBannerAd(bannerPlacementInputField.text, true, size, screenPos);
+        _bannerAd.WillAppear += WillAppearBanner;
+        _bannerAd.DidClick +=DidClickBanner;
+        _bannerAd.DidRecordImpression += DidRecordImpressionBanner;
+        _bannerAd.DidDrag += DidDragBanner;
+
+        var keywords = _bannerAd.Keywords ??= new Dictionary<string, string>();
+        keywords.Add("bnr_keyword1", "bnr_value1"); 
+        keywords.Add("bnr_keyword2", "bnr_value2");
+        _bannerAd.Keywords = keywords;
+
+        _bannerAd.VerticalAlignment = (ChartboostMediationBannerVerticalAlignment)verticalAlignmentDropdown.value;
+        _bannerAd.HorizontalAlignment = (ChartboostMediationBannerHorizontalAlignment)horizontalAlignmentDropdown.value;
+        
+        // TODO: Debug mode (Remove)
+        var image =_bannerAd.gameObject.AddComponent<Image>();
+        image.color = new Color(0.25f,1,0.25f, .25f);
```

Comment:
Is this going to be removed?

Reply:
Yes it will be removed once most of my testing is done